frameworks/keyed/incremental-dom/src/components/table.ts

```diff
@@ -0,0 +1,46 @@
+import { elementOpen, elementClose, close, text, open, attr, applyAttrs, applyStatics } from 'incremental-dom';
+import { DataEntry } from '../data-collection';
+
+
+export function Table(func: () => void) {
+    elementOpen('table', null, null, 'class', 'table table-hover table-striped test-data');
+    elementOpen('tbody', null, null, 'id', 'tbody');
+    func();
+    elementClose('tbody');
+    elementClose('table');
+}
+
+function TableCell(className: string, func: () => void) {
+    elementOpen('td', null, null, 'class', className);
+    func();
+    close();
+}
+
+export function TableRow(data: DataEntry, selected: boolean, onRowClick: () => void, onRowDelete: () => void) {
+    const row = open('tr', data.id);
+    attr('class', selected ? 'danger' : '');
+    applyAttrs();
+    if (!row.__click) {
+        row.addEventListener('click', onRowClick);
```

Comment:
You can use IDOM to assign click handlers by setting the attribute 'onclick'

Reply:
Done!